husky_components/ckeditor/main.js

```diff
@@ -148,15 +136,10 @@ define([], function() {
             delete config._ref;
             delete config.require;
             delete config.element;
-            delete config.linksEnabled;
+            delete config.linkEnabled;
             delete config.tableEnabled;
-            delete config.scriptEnabled;
-            delete config.iframeEnabled;
             delete config.maxHeight;
```

Comment:
Looks like this have been like this before, but in general it is not a good idea to delete properties, since you get slow objects then.